packages/container/lib/container.js

```diff
@@ -190,6 +190,13 @@ function wrapManagerInDeprecationProxy(manager) {
   if (HAS_NATIVE_PROXY) {
     let validator = {
       get(obj, prop) {
+        // make work with node's formatValue
+        if (!(prop in obj)) { return ; }
```

Comment:
this will obviously now fail if any of the following are actually defined on `obj`

*  `Symbol(util.inspect.custom)`
* `Symbol(Symbol.toStringTag)`